README.md

```diff
@@ -1,20 +1,38 @@
-# Accessibility Check Action
+# accessibility-check-action
 
-In order for Stark to scan your GitHub web repository, you’ll need to set up a custom workflow. This workflow enables Stark to scan your web repository for accessibility issues.
+## Table of Contents
 
-## Setting up your repository
+- [Basic Usage](#basic-usage)
+- [Usage with Stark Projects](#usage-with-stark-projects)
+- [More Examples](#more-examples)
+- [Technical Details](#technical-details)
 
-First, create a Github workflow file at `.github/workflows/starkflow.yml`. (The file name is important: that’s how Stark finds the correct workflow to run.)
+## Basic Usage
+
+Basic usage of the action is exactly as shown in the next section's template, except you don't need to provide the action a `token`.
+
+## Usage with Stark Projects
+
+### Setting up a repository's workflow
+
+To use Stark Projects to scan your Github repositories for accessibility issues, you’ll need to set up custom workflows in those repositories.
+
+Begin by creating a GitHub workflow file at `.github/workflows/starkflow.yml`. Using the right file path and name is important, otherwise Stark won't be able to scan your repository.
+
+Next, use the following template for your workflow, updating the appropriate values as indicated:
 
-Next, copy and paste  the following template (update the necessary defaults and remove empty params):
 ```yml
 name: Stark Web Audit
+run-name: ${{ github.event.inputs.display-title }}
 
 on:
   workflow_dispatch:
     inputs:
       token:
-        description: 'Stark Token'
+        description: 'Stark token'
+        required: true
+      display-title:
+        description: 'Display title'
```

Comment:
Anything we need to add here to say what this does or where it shows up? Or perhaps just a better default?

Reply:
Hmm good question. These parameters allow us to send input to the workflow when we run it. In this case `display-title` lets the Stark backend to assign a specific title to a particular scan, which helps with identification (and other stuff if we decide to take advantage down the line.) All that to say that a default sort of defeats the point, but I'll add a clarifying comment!